src/components/DraggableViewSection.tsx

```diff
@@ -33,54 +56,88 @@ function DraggableViewSection({ children, offset, dimension, vertical, dimension
     setHasLongHovered(false);
   }, []);
 
-  const handleDrag = useCallback((event: DragEvent<HTMLDivElement>) => {
-    if (!(event.clientX === 0 && event.clientY === 0)) {
-      if (prevDragPosition.current) {
-        const dimensionChange = vertical 
-          ? event.clientY - prevDragPosition.current.y
-          : event.clientX - prevDragPosition.current.x;
-        onDimensionChange(dimensionChange);
-      }
-      prevDragPosition.current = { x: event.clientX, y: event.clientY }
-    }
-  }, [onDimensionChange, vertical]);
+  const handleDrag = useCallback(
+    (_: DragEvent<HTMLDivElement | Body>) => {
+      setTimeout(() => {
+        if (
+          dragEvent &&
+          !(
+            dragEvent.current?.clientX === 0 && dragEvent.current?.clientY === 0
```

Comment:
This line is unnecessary because the two conditions below are false when clientX and clientY are zero